src/scancode/cli.py

```diff
@@ -1202,6 +1202,10 @@ def display_summary(codebase, scan_names, processes, verbose):
                                 '%(final_size_count)s' % locals())
 
     echo_stderr('Timings:')
+
+    timestamp = codebase.scan_start
+    echo_stderr('  timestamp: %(timestamp)s'% locals())
```

Comment:
Could we call this Scan start instead here and in the JSON output? ... since this is what this is.